src/nifti-extension.ts

```diff
@@ -0,0 +1,33 @@
+export class NiftiExtension {
```

Comment:
breaking: casing changed from `NIFTIEXTENSION` to `NiftiExtension`

TODO: `NIFTIExtension` would be more consistent